website/src/_posts/2019-04-1.0.md

```diff
@@ -89,10 +89,19 @@ Here are some highlights on what we’ve managed to delived in the Uppy Month:
 - Works great on mobile
 - Works great with Transloadit. Works great without
 
-- knows about encoding
+We also left two features "behind":
+
+> Offering sugared shortcuts for novice users (presets) 
+
+We felt Uppy is straightforward enough to use, presets could end up making this less clear. Robodog does offer less boilerplatey integrations, but this was done as a wrapper instead of a preset.
+
+> Themable UI with a beautiful default 
+
+Instead of going for `{ theme: 'dark' }`, we only offer theming via CSS. It is of course very feasible to create a `dark.css` file, and maybe we can come up with a section on the site for easily sharing and distributing such files, but we let go of this desire for 1.0.
+
+It's not :100:, but for a 3 year gameplan, we are very happy with this result!
 
```

Comment:
Not sure if we even need this section, we could then as well say we don’t have Box and Google Photos support. Yeah, I know those things were on the list on the website, but.